src/dialog/test/dialog.spec.ts

```diff
@@ -800,6 +800,29 @@ describe('ThyDialog', () => {
         expect(dialogContainer.animationState).toBe('exit');
     });
 
+    it('should has "pointer-event: none" at dialog-container when animation phaseName equal start and toState equal exit', fakeAsync(() => {
+        const dialogRef = dialog.open(DialogSimpleContentComponent, { viewContainerRef: testViewContainerRef });
+        const dialogContainer = viewContainerFixture.debugElement.query(By.directive(ThyDialogContainerComponent)).componentInstance;
+
+        expect((getDialogContainerElement() as HTMLElement).style.pointerEvents).toEqual('');
+
+        dialogContainer.onAnimationStart({
+            phaseName: 'start',
+            toState: 'exit'
+        });
+        viewContainerFixture.detectChanges();
+
+        expect((getDialogContainerElement() as HTMLElement).style.pointerEvents).toEqual('none');
+
+        tick(150);
+        dialogContainer.onAnimationStart({
+            phaseName: 'end',
```

Comment:
phaseName：the animation phase in which the callback was invoked, one of "start" or "done".